wordpress-comments/src/main/java/org/wordpress/aztec/plugins/wpcomments/toolbar/MoreToolbarButton.kt

```diff
@@ -45,6 +46,10 @@ class MoreToolbarButton(val visualEditor: AztecText) : IToolbarButton {
     }
 
     override fun matchesKeyShortcut(keyCode: Int, event: KeyEvent): Boolean {
+        if (DeviceUtils.getInstance().isChromebook(context)) {
```

Comment:
Should we add an alternative shortcuts for Chrome OS? This is usually what other apps do when there is a conflict with a specific OS or another app.

Reply:
> Should we add an alternative shortcuts for Chrome OS? This is usually what other apps do when there is a conflict with a specific OS or another app.